web/client/plugins/Swipe.jsx

```diff
@@ -19,17 +19,24 @@ import { createPlugin } from '../utils/PluginsUtils';
 
 import SwipeSettings from './SwipeSettings';
 import SliderSwipeSupport from '../components/map/openlayers/swipe/SliderSwipeSupport';
+import SpyGlassSupport from '../components/map/openlayers/swipe/SpyGlassSupport';
 
-const Support = ({ map, layer, active }) => {
-    return <SliderSwipeSupport map={map} layer={layer} active={active} />;
+export const Support = ({ mode, map, layer, active, radius, sliderType }) => {
+    if (mode === "spy") {
+        return <SpyGlassSupport map={map} layer={layer} active={active} radius={radius} />;
+    }
+    return <SliderSwipeSupport map={map} layer={layer} active={active} type={sliderType} />;
 };
 
 const swipeSupportSelector = createSelector([
     getSelectedLayer,
     layerSwipeSettingsSelector
 ], (layer, swipeSettings) => ({
     layer: layer?.id,
-    active: swipeSettings.active || false
+    active: swipeSettings.active || false,
+    mode: "spy", // TODO: use UI configuration
```

Comment:
Just keep swipe for the moment
```suggestion
    mode: "swipe", // TODO: use UI configuration
```